### ISSUE TITLE
J2534 doesn't connect with car in test mode

### ISSUE THREAD

**Issue by AndrewLitchfield** [Comer352L/FreeSSM#42]
A very minor observation.  When using the latest branch and Tactrix to connect to the car FreeSSM cannot connect if a 2005-2009 Legacy has the green test mode connectors made.  Using a pass thru connection works well but I'm now really happy with the speed of the J2534 connection to the ECM :)  Screenshot attached.

![freessm-dev](https://user-images.githubusercontent.com/45085538/80862054-68547480-8cc6-11ea-967b-65f2aa5b3b24.png)

**Comment by paladinshiva** [comment]
Why do you need green connectors connected???

**Comment by Comer352L** [comment]
That's weird...
Do I understand right, that you were testing with
a) a serial pass-through interface (=> ISO-14230) and
b) a J2534-interface (=> ISO-15765 CAN)
with the same car under the same conditions ?
Sure ?

**Comment by AndrewLitchfield** [comment]
Yes, with pass-thru FreeSSM stopped auto test mode and allowed me to run the tests listed in FreeSSM.  WIth the Tactrix connection FreeSSM couldn't stop auto test mode.  I repeated this twice to be sure.
